code/modules/gear_presets/_select_equipment.dm

```diff
@@ -662,6 +662,36 @@ GLOBAL_LIST_INIT(rebel_rifles, list(
 				new_human.equip_to_slot_or_del(new /obj/item/storage/belt/marine, WEAR_WAIST)
 		spawn_weapon(gunpath, ammopath, M, 0, ammo_amount)
 
+/datum/equipment_preset/proc/spawn_merc_elite_weapon_lowthreat(atom/M, ammo_amount = 12, shotgun_chance = 50, spawn_belt = 1)
```

Comment:
why is this defined so high up instead of on `/datum/equipment_preset/other/elite_merc`

Reply:
also given you're only piping humans into this, you can just type it appropriately